content/ja/handbook/engineering/infrastructure-platforms/data-access/help.md

```
@@ -0,0 +1,138 @@
+   - Omnibus/Charts のパッケージ化された Postgres に関連する問題については [#g_distribution](https://gitlab.enterprise.slack.com/archives/C1FCTU4BE) に投稿します（内部）
+
+### 1. 開始
+
+緊急事態が以下のような設定または操作に関連する場合:
+
+- Postgres の運用に関連する緊急事態
+- 接続エラー
+- レプリケーションエラー
+- SSL エラー
+
+ステップ [2. 設定エラー](#2-configuration-or-operational-errors) に進んでください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)